include/micm/solver/backward_euler.inl

```diff
@@ -194,13 +187,13 @@ namespace micm
           n_successful_integrations = 0;
           H *= 2.0;
         }
+        // Don't let H go past the time step
+        H = std::min(H, time_step - t);
```

Comment:
I think this check is needed for both `if` and `else` blocks, so it should be moved to its original place?

Reply:
well, the if block reduces the time step (even though it's multiplying something), but there's no harm in me moving it back to how it was